aiohttp_jinja2/__init__.py

```diff
@@ -24,6 +24,8 @@ def setup(app, *args, app_key=APP_KEY, context_processors=(),
         env.globals.update(GLOBAL_HELPERS)
     if filters is not None:
         env.filters.update(filters)
+    if 'autoescape' not in kwargs:
+        env.autoescape = aiohttp_jinja2_autoescape
```

Comment:
Why not autoescape everything by default?
Why do you choose based on template template suffix?